modules/mirror-registry-localhost-update.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+[id="mirror-registry-localhost-update_{context}"]
+= Updating mirror registry for Red Hat OpenShift from a local host 
+
+This procedure explains how to update the _mirror registry for Red Hat OpenShift_ from a local host using the `upgrade` command. Updating to the latest version ensures bug fixes and security vulnerability fixes. 
+
+[IMPORTANT]
+====
+When updating, there is intermittent downtown of your mirror registry, as it is restarted during the update process. 
+====
+
+.Prerequisites
+
+* You have installed the _mirror registry for Red Hat OpenShift_ on a local host. 
+
+.Procedure 
+
+* To upgrade the the _mirror registry for Red Hat OpenShift_ from localhost, enter the following command:
++
+[source,terminal]
+----
+$ sudo ./mirror-registry upgrade -v 
+----
++
+[NOTE]
+====
+* Users who upgrade the _mirror registry for Red Hat OpenShift_ with the `./mirror-registry upgrade` flag must include the same credentials used when creating their mirror registry. For example, if you installed the _mirror registry for Red Hat OpenShift_ with `--quayHostname <host_example_com>` and `--quayRoot <example_directory_name>`, you must include that string to properly upgrade the mirror registry.
+====
```

Comment:
Way out of scope for this PR, but if you are attaching more than one block of text with a list continuation, you can wrap the content in an open block. This prevents you from having to use multiple list continuations.

In this crude example, everything after the first `+`  would be nested under your first step [1]
```
+
--
---
$
---
[NOTE]
====
Text
====
--
```
[1] https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation

Reply:
Awesome. Thanks for sharing!